<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="ButtonBackground">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>0</y>
      <width>1451</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="ModeBackground">
    <property name="geometry">
     <rect>
      <x>1326</x>
      <y>551</y>
      <width>71</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>630</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>630</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>651</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>680</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>867</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="TriangleButton">
       <property name="text">
        <string>Triangle</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Figure</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="HexagonButton">
       <property name="text">
        <string>Hexagon</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Figure</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="SqauerButton">
       <property name="text">
        <string>Square</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Figure</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RectangleButton">
       <property name="text">
        <string>Rectangle</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Figure</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="PolylineButton">
       <property name="text">
        <string>Polyline</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Figure</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RhombusButton">
       <property name="text">
        <string>Rhombus</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Figure</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Star5Button">
       <property name="text">
        <string>Star5</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Figure</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Star6Button">
       <property name="text">
        <string>Star6</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Figure</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Star7Button">
       <property name="text">
        <string>Star7</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Figure</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Star8Button">
       <property name="text">
        <string>Star8</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Figure</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="CircleButton">
       <property name="text">
        <string>Circle</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Figure</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>650</y>
      <width>65</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="DrawButton">
       <property name="text">
        <string>Draw</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Mode</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="MoveButton">
       <property name="text">
        <string>Move</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Mode</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>630</y>
      <width>68</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Center:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Square:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Perimetr:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>560</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="LeftRotateButton">
       <property name="text">
        <string>←</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Rotate</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RightRotateButton">
       <property name="text">
        <string>→</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Rotate</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>590</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="PlusButton">
       <property name="text">
        <string>+</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Size</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="MinusButton">
       <property name="text">
        <string>-</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Size</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="ClearButton">
    <property name="geometry">
     <rect>
      <x>1007</x>
      <y>0</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(87, 227, 137);
color : rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>622</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Modes:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>871</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show point
(for circle)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPaint_100D_Pro_Max">
    <property name="title">
     <string>Paint 100D Pro Max</string>
    </property>
   </widget>
   <addaction name="menuPaint_100D_Pro_Max"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="Rotate"/>
  <buttongroup name="Mode"/>
  <buttongroup name="Figure"/>
  <buttongroup name="Size"/>
 </buttongroups>
</ui>
